server/container/guice/jmx/src/main/java/org/apache/james/modules/server/JMXServerModule.java

```diff
@@ -23,15 +23,7 @@
 import java.util.List;
 
 import org.apache.commons.configuration.ConfigurationException;
-import org.apache.james.adapter.mailbox.MailboxCopierManagement;
-import org.apache.james.adapter.mailbox.MailboxCopierManagementMBean;
-import org.apache.james.adapter.mailbox.MailboxManagerManagement;
-import org.apache.james.adapter.mailbox.MailboxManagerManagementMBean;
-import org.apache.james.adapter.mailbox.MailboxManagerResolver;
-import org.apache.james.adapter.mailbox.QuotaManagement;
-import org.apache.james.adapter.mailbox.QuotaManagementMBean;
-import org.apache.james.adapter.mailbox.ReIndexerManagement;
-import org.apache.james.adapter.mailbox.ReIndexerManagementMBean;
+import org.apache.james.adapter.mailbox.*;
```

Comment:
Here too, you need to revert the wildcard import.

Reply:
intellij :(